src/code_8092334.c

```diff
@@ -1,15 +1,8 @@
+#include "code_8092334.h"
```

Comment:
Shouldn't this be under `global.h`?

Reply:
ah whoops, that would fix both problems yeah